Dockerfile

```diff
@@ -1,19 +1,33 @@
-FROM tensorflow/tensorflow:2.1.0-gpu-py3
+FROM tensorflow/tensorflow:2.4.2-gpu
```

Comment:
Why forcing GPU?
As I remember the base tensorflow package will use a GPU if configured so, and otherwise will revert to CPU. Is there a specific need to force this package flavor?

Reply:
I don't use GPU on my machine. It works well that way on a CPU-only machine.